common/src/main/java/ch/admin/bag/covidcertificate/common/faq/FaqItem.kt

```diff
@@ -29,16 +31,18 @@ data class HeaderItem(val header: Header) : FaqItem() {
 	}
 
 	override fun bindView(view: View, onItemClickListener: (() -> Unit)?) {
-		view.findViewById<TextView>(R.id.item_faq_header_title).text = header.title
-		view.findViewById<TextView>(R.id.item_faq_header_text).apply {
+		val binding = ItemFaqHeaderBinding.bind(view)
```

Comment:
No, because the binding is only local to this method, therefore no strong reference is kept. And also, ViewHolders are destroyed when the fragment view is destroyed, they dont survive configuration changes.